sos/report/plugins/containers_common.py

```diff
@@ -30,27 +30,43 @@ def setup(self):
             '/etc/subuid',
             '/etc/subgid',
         ])
-        self.add_cmd_output(['loginctl user-status'])
 
         users_opt = self.get_option('rootlessusers')
         users_list = []
         if users_opt:
             users_list = [x for x in users_opt.split(':') if x]
 
         user_subcmds = [
-            'info',
-            'unshare cat /proc/self/uid_map',
-            'unshare cat /proc/self/gid_map'
+            'podman info',
+            'podman unshare cat /proc/self/uid_map',
+            'podman unshare cat /proc/self/gid_map',
+            'podman images',
+            'podman pod ps',
+            'podman port --all',
+            'podman ps',
+            'podman ps -a',
+            'podman stats --no-stream --all',
+            'podman version',
+            'podman volume ls',
+            'buildah info',
+            'buildah unshare cat /proc/self/uid_map',
+            'buildah unshare cat /proc/self/gid_map',
+            'buildah containers',
+            'buildah containers --all',
+            'buildah images',
+            'buildah images --all',
+            'buildah version',
         ]
         for user in users_list:
             # collect user's containers' config
             self.add_copy_spec(
                 '%s/.config/containers/' % (os.path.expanduser('~%s' % user)))
-            # collect the user's podman/buildah info and uid/guid maps
-            for binary in ['/usr/bin/podman', '/usr/bin/buildah']:
-                for cmd in user_subcmds:
-                    self.add_cmd_output([
-                        'machinectl -q shell %s@ %s %s' % (user, binary, cmd)
-                    ], foreground=True)
+            # collect user-status
+            self.add_cmd_output(['loginctl user-status %s' % user])
+            # collect the user's related commands
+            for cmd in user_subcmds:
+                self.add_cmd_output([
+                    'machinectl -q shell %s@ /usr/bin/%s' % (user, cmd)
+                ], foreground=True)
```

Comment:
Same here, with this style this doesn't need to be a list. Or it could be refactored to something like

```python
self.add_cmd_output([
    "machinectl -q shell %s@ /usr/bin/%s" % (user, cmd) for cmd in user_subcmds
], foreground=True)
```